sklearn/discriminant_analysis.py

```diff
@@ -143,9 +143,6 @@ class LinearDiscriminantAnalysis(BaseEstimator, LinearClassifierMixin,
     .. versionadded:: 0.17
        *LinearDiscriminantAnalysis*.
 
-    .. versionchanged:: 0.17
-       Deprecated :class:`lda.LDA` have been moved to :class:`LinearDiscriminantAnalysis`.
```

Comment:
Hm so the question is do we document when the rename happened in the renamed class, right?  I guess the ``versionadded`` tag above is enough and so I think it's fine to remove the versionchanged.